package.json

```diff
@@ -511,6 +655,31 @@
             "type": "string"
           }
         },
+        "r.rMarkdownCodeLens.option": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "r.runCurrentChunk",
+            "r.runAboveChunks",
+            "r.runCurrentAndBelowChunks",
+            "r.runAllChunks",
+            "r.goToPreviousChunk",
+            "r.goToNextChunk"
+          ],
+          "description": "Available options:\n\nr.selectCurrentChunk\nr.runCurrentChunk\nr.runAboveChunks\nr.runCurrentAndBelowChunks\nr.runBelowChunks\nr.runAllChunks\nr.runPreviousChunk\nr.runNextChunk\nr.goToBeginningOfChunk\nr.goToEndOfChunk\nr.goToPreviousChunk\nr.goToNextChunk\nr.goToFirstChunk\nr.goToLastChunk\n\n----------------------------------\nCustomize both the options AND its orders (that is, code lens respect user-specified orders):"
+        },
+        "r.rMarkdownCodeLens.enableCodeLens": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable RMarkdown code lens"
+        },
+        "r.rMarkdownCodeLens.chunkBackgroundColor": {
```

Comment:
Here it seems to make more sense to use `r.rmarkdown.chunkBackgroundColor` in this case as the background color of chunks is handled by document decoration rather than being part of CodeLens, i.e. it has nothing to do with CodeLens but it part of settings of R Markdown document rendering.

Reply:
Thanks for the review. Renamed to `r.rmarkdown.chunkBackgroundColor` and made the customization of CodeLens and chunk background color independent